sitewit/models.py

```diff
@@ -113,3 +119,30 @@ def delete(cls, account_token):
         result = cls.get_service().delete_account(account_token)
 
         return Account(result)
+
+
+class Subscription(SiteWitServiceModel):
+    def __init__(self, site_id, url, data):
+        self.site_id = site_id
+        self.url = url
+        self.ad_spend = Decimal(data['budget'])
+        self.price = Decimal(data['fee'])
+        self.campaign_id = data['campaignId']
+        self.currency = data['currency']
+
+    @classmethod
+    def iter_subscriptions(cls):
+        service = cls.get_service()
+        limit = 100
+
+        for offset in itertools.count(0, limit):
```

Comment:
As for me it's not about statements, but more about logic.

This:

> Query data until chunk is incomplete and update offset after each time.

Seems more readable to me than this:

> For each offset starting from 0 with step N I will query chunks, and will stop once chunk is incomplete.